HunterPie.Native/Core/Managers/Damage/manager.cpp

```diff
@@ -22,17 +22,19 @@ DamageTrackManager* DamageTrackManager::GetInstance()
     return m_Instance;
 }
 
-void DamageTrackManager::UpdateDamage(EntityDamageData damageData)
+void DamageTrackManager::UpdateDamage(const EntityDamageData& damageData)
 {
-    intptr_t& target = damageData.target;
+    intptr_t target = damageData.target;
 
     if (m_Trackings.find(target) == m_Trackings.end())
         m_Trackings.insert({ target, new HuntStatistics() });
 
     HuntStatistics*& statistics = m_Trackings.at(target);
 
-    if (damageData.source.index >= std::size(statistics->entities) || damageData.source.index < 0)
+    if (damageData.source.index >= std::size(statistics->entities) || damageData.source.index < 0) {
+        LOG("Invalid source entity index��%d.", damageData.source.index);
```

Comment:
I think the string encoding broke there `index£º` LOL

Reply:
I think I've typed a full-width colon by accident, encoded in GB2312 🌚
![image](https://user-images.githubusercontent.com/11611687/189608130-c69ccf6e-b418-494b-ab35-0eda7cd6f2fc.png)

VS IDE seems won't bother choosing utf8nobom for cpp files.